src/ClientData/include/ClientData/ScopeCollection.h

```diff
@@ -0,0 +1,45 @@
+
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CLIENT_DATA_SCOPE_COLLECTION_H_
```

Comment:
I don't like `ScopeCollection`. I would say that this class should include "durations" in its name. It's going to be mostly stats about durations, or am I wrong?

Reply:
Sure. What about ScopeStatsCollection? I think it's a bit more accurate since ScopeStats is already well known & it holds everything related to this.